charters/PipelineExecutionService/charter.md

```diff
@@ -0,0 +1,63 @@
+# [Pipeline Execution Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Pipeline Execution Service executes data processing pipelines containing analysis tasks performed on scientific data in the Data Coordination Platform (DCP). The outputs of these pipelines are used for scientific analysis and interpretation.  
+
+## Objectives
+
+The Pipeline Execution Service runs pipelines reliably and at-scale for the DCP. The team implementing the service also operates the pipeline infrastructure for the Human Cell Atlas project. 
+
+## Definitions
+
+__pipeline:__ A collection of one or more functional tasks that operate on input data and, from that, make an output that is transformed input data, or derive features often used to interpret the input data. In a high throughput setting, these tasks are often automated to be performed in a batch setting.
```

Comment:
One suggestion on wording: I think changing "make an output that is transformed input data" to "transform the input data" makes this sentence easier to understand

Reply:
added